tests/ops/fixtures/saas/hubspot_fixtures.py

```diff
@@ -133,22 +135,41 @@ def hubspot_erasure_data(
             }
         }
     )
+    updated_headers, contacts_request_formatted_body = format_body(
+        {}, contacts_request_body
+    )
 
-    updated_headers, formatted_body = format_body({}, body)
-
-    # create contact
     contacts_request: SaaSRequestParams = SaaSRequestParams(
         method=HTTPMethod.POST,
         path=f"/crm/v3/objects/contacts",
         headers=updated_headers,
-        body=formatted_body,
+        body=contacts_request_formatted_body,
     )
     contacts_response = connector.create_client().send(contacts_request)
     contacts_body = contacts_response.json()
     contact_id = contacts_body["id"]
 
-    # no need to subscribe contact, since creating a contact auto-subscribes them
+    # create user
+    users_request_body = json.dumps(
+        {
+            "email": hubspot_erasure_identity_email,
+        }
+    )
+    updated_users_request_headers, users_request_formatted_body = format_body(
+        {}, users_request_body
+    )
 
+    users_request: SaaSRequestParams = SaaSRequestParams(
```

Comment:
Same as the other comment about not using the SaaSConnector to make the API calls.

Reply:
done!